bin/compile

```diff
@@ -15,13 +15,17 @@ S3_BUCKET=${HEROKU_GEO_BUILDBACK_S3_BUCKET:-"heroku-buildpack-geo"}
 
 # Parameters
 BUILD_DIR=$1
+BP_DIR=$(cd $(dirname ${0:-}); cd ..; pwd)
```

Comment:
Is it really necessary here?
Same things is in line 20

Reply:
It seems this got screwed when rebasing. I removed the duplicate line. Good catch 👍